modules/core/shared/src/main/scala/coursier/maven/MavenRepositoryLike.scala

```diff
@@ -0,0 +1,21 @@
+package coursier.maven
+
+import coursier.core.{Authentication, Module, Repository}
+import coursier.util.Artifact
+
+trait MavenRepositoryLike extends Repository {
```

Comment:
If this is meant to be user API, some Scaladoc would be nice. What make a repository Maven like?

Reply:
Yeah, good point. Just added some scaladoc to it.